packages/react/src/components/Switch/Switch.js

```diff
@@ -112,7 +113,7 @@ Switch.propTypes = {
 
 Switch.defaultProps = {
   selected: false,
-  text: 'Provide text',
+  text: FeatureFlags.enabled('enable-v11-release') ? null : 'Provide text',
```

Comment:
This change might not be relevant with the Content Switcher refactor, potentially, so you might be able to leave it alone since this should be a `children` prop anyways